examples/pom.xml

```diff
@@ -83,5 +83,11 @@
         <module>spring-boot-with-gradle-on-openshift-example</module>
       </modules>
     </profile>
+    <profile>
+      <id>with-helm-examples</id>
+      <modules>
+        <module>helm-example</module>
+      </modules>
+    </profile>
```

Comment:
This is because I'm thinking to use HELM in real, but if this is too complex, I will remove it and only verify the artifacts are generated.